jgrapht-ext/src/test/java/org/jgrapht/ext/CSVExporterTest.java

```diff
@@ -0,0 +1,167 @@
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* ------------------------------
+ * CSVExporterTest.java
+ * ------------------------------
+ * (C) Copyright 2014, by Ivan Gavrilovic.
+ *
+ * Original Author:  Ivan Gavrilovic
+ *
+ *
+ */
+package org.jgrapht.ext;
+
+import junit.framework.TestCase;
+import org.jgrapht.DirectedGraph;
+import org.jgrapht.UndirectedGraph;
+import org.jgrapht.experimental.dag.DirectedAcyclicGraph;
+import org.jgrapht.graph.DefaultDirectedGraph;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.SimpleGraph;
+import org.jgrapht.imp.CSVImporter;
+import org.jgrapht.imp.StringVertexParser;
+import org.jgrapht.util.VertexPair;
+
+import java.io.FileInputStream;
+import java.io.StringWriter;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Testing the CSV export.
+ * @author Ivan Gavrilović
+ */
+public class CSVExporterTest extends TestCase {
+    public void testExportCSV() {
+        DirectedGraph<String , DefaultEdge> directed = new DefaultDirectedGraph<String, DefaultEdge>(DefaultEdge.class);
+        UndirectedGraph<String, DefaultEdge> undirected = new SimpleGraph<String, DefaultEdge>(DefaultEdge.class);
+
+        directed.addVertex("\"A"); undirected.addVertex("A");
+        directed.addVertex("B\n"); undirected.addVertex("B");
+        directed.addVertex("C"); undirected.addVertex("C");
+        directed.addVertex("D"); undirected.addVertex("D");
+
+        directed.addEdge("\"A", "B\n"); undirected.addEdge("A", "B");
+        directed.addEdge("\"A", "D"); undirected.addEdge("A", "D");
+        directed.addEdge("B\n", "D"); undirected.addEdge("B", "D");
+
+        String dir =    "\"\"\"A\",\"B\n\"\n" +
+                        "\"\"\"A\",D\n" +
+                        "\"B\n\",D\n" +
+                        "C\n" +
+                        "D";
+        String undir =  "A,B\n" +
+                        "A,D\n" +
+                        "B,A\n" +
+                        "B,D\n" +
+                        "C\n" +
+                        "D,A\n" +
+                        "D,B";
+
+        StringWriter wDir = new StringWriter();
+        StringWriter wUnDir = new StringWriter();
+        CSVExporter<String, DefaultEdge> csv = new CSVExporter<String, DefaultEdge>();
+
+        try {
+            csv.export(directed, wDir);
+            csv.export(undirected, wUnDir);
+            assertEquals(dir, wDir.toString());
+            assertEquals(undir, wUnDir.toString());
+        }
+        catch (Exception e){
+            e.printStackTrace();
+            fail();
+        }
+    }
+
+    public void testImport(){
+        String paths[] = new String[]{
+                "/Users/ivan/Projects/ontology/activity-subsumer/src/test/resources/word_net_nouns_hyponym",
```

Comment:
These paths seem to refer to some local directory of yours? Please revise such that it works outside your system as well.